libcxx/test/libcxx/utilities/memory/util.smartptr/util.smartptr.shared/libcxx.control_block_layout.pass.cpp

```diff
@@ -195,10 +195,12 @@ int main(int, char**) {
   test<TrivialEmptyType, FinalEmptyAlloc>();
   test<TrivialEmptyType, NonTrivialAlloc>();
 
+#if !defined(_LIBCPP_HAS_NO_LIBRARY_ALIGNED_ALLOCATION)
```

Comment:
AFAIK In tests we shouldn't use the `_LIBCPP` prefixed macros. There should be another test specific macro, defined in: https://github.com/llvm/llvm-project/blob/main/libcxx/test/support/test_macros.h